src/template/configSchema.json

```diff
@@ -82,6 +82,19 @@
                             "$ref": "#/definitions/default:array"
                         }
                     ]
+                }
+            },
+            "additionalProperties": false
+        },
+        "section:required": {
+            "title": "Required section",
+            "description": "Required section of help",
+            "properties": {
+                "names": {
+                    "$ref": "#/definitions/section:optional/properties/names"
```

Comment:
I suggest to move following properties to `root` lvl of definitions: `names` and `postfix`